skorch/tests/test_scoring.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import numpy as np
+import torch
+
+
+class TestLossScoring:
+    @pytest.fixture(scope="module")
+    def data(self, classifier_data):
+        return classifier_data
+
+    @pytest.fixture(scope="module")
+    def val_data(self, classifier_data):
```

Comment:
I think for testing this function, we don't actually need separate validation data, we can just use the training data.